common/app/views/fragments/immersiveVideo.scala.html

```diff
@@ -0,0 +1,28 @@
+@(page: ContentPage)(implicit request: RequestHeader)
```

Comment:
Maybe this file could be called `heroicVideo.scala.html` for consistency with the class names?